Лабораторная работа 4/task1.py

```diff
@@ -0,0 +1,36 @@
+# TODO решить с помощью list comprehension и распечатать его
+import pprint
+
```

Comment:
`pprint.pprint([{'bin': bin (i), 'dec': i, 'hex': hex (i), 'oct': oct (i)} for i in range(16)])`